AirSkyBoat/AirSkyBoat
# LSB Cherry-pick: Kill ZONE_TYPE::BATTLEFIELD

**_I affirm:_**


## Please enter a player-facing description
Nothing will change


## What does this pull request do? (Please be technical)
ZONE_TYPE::BATTLEFIELD has now been deleted from the codebase. All mobs that are in BCNMS should have MOBTYPE_BATTLEFIELD inside its sql in order for it to work.
- https://github.com/LandSandBoat/server/pull/3884


## Steps to test these changes
We need to go into EVERY BCNM and make sure the mobs attack / are engageable

BCNM Check List:
- [x] Bearclaw_Pinnacle
- [ ] Boneyard_Gully
- [ ] The_Shrouded_Maw
- [ ] Mine_Shaft_2716
- [ ] Spire_of_Holla
- [ ] Spire_of_Dem
- [ ] Spire_of_Mea
- [ ] Spire_of_Vahzl
- [ ] Monarch_Linn
- [ ] Empyreal_Paradox
- [ ] Temenos (fuck us)
- [ ] Apollyon (fuck us again)
- [ ] Horlais_Peak
- [ ] Waughroon_Shrine
- [x] Balgas_Dais
- [ ] Sacrificial_Chamber
- [ ] Throne_Room
- [ ] Chamber_of_Oracles
- [ ] Full_Moon_Fountain
- [ ] Stellar_Fulcrum
- [ ] LaLoff_Amphitheater
- [ ] The_Celestial_Nexus
- [ ] Walk_of_Echoes
- [x] QuBia_Arena
- [x] Cloister_of_Gales
- [x] Cloister_of_Storms
- [x] Cloister_of_Frost
- [x] Cloister_of_Flames
- [x] Cloister_of_Tremors
- [x] Cloister_of_Tides
- 

## Special Deployment Considerations
If we miss a mobs mobtype in the sql the bcnm won't work.
